examples/complete/main.tf

```diff
@@ -1,11 +1,13 @@
 locals {
-  project_id   = "<your-project-id>"
-  network      = "<your-network>"
-  subnetwork   = "<your-subnetwork>"
-  region       = "<your-region>"
-  zone         = "<your-zone>"
-  domain       = "<example.com>"
-  managed_zone = "<your-managed-zone>"
+  project_id                   = "<your-project-id>"
+  region                       = "<your-region>"
+  zone                         = "<your-zone>"
+  domain                       = "<example.com>"
+  managed_zone                 = "<your-managed-zone>"
+  enable_oslogin               = false
+  google_logging_enabled       = true
+  google_logging_use_fluentbit = false
```

Comment:
> Can we remove the locals from all the examples, as these are optional variables anyways and now add extra entries to the list in the example.

Reply:
I added the locals to make sure the user understand the choices are deliberate and can be changed ONLY looking at the locals section without altering the bulk of the code past the locals.